services/browsers/database.go

```diff
@@ -35,7 +35,7 @@ type Browser struct {
 	Name         string   `json:"name"`
 	Timestamp    uint64   `json:"timestamp"`
 	Dapp         bool     `json:"dapp?"`
-	HistoryIndex int      `json:"history-index,omitempty"`
+	HistoryIndex *int     `json:"history-index,omitempty"`
```

Comment:
Wouldn't it be enough to remove `omitempty`? Or we still need to distinguish between `0` and no history index?

Reply:
i think it is correct to distinguish between 0 and no history. history index should be consistent with the history array. e.g. if there is no array or no items in array - history index 0 is confusing

Reply:
In theory, no `history` might be considered that there is no history instead of looking at the index. I guess that would be the same? Just `*int` looks odd in this case but all is correct :)

Reply:
i guess it makes sense, a consumer will have to double check that index is not out of bounds anyway